### ISSUE TITLE
Decoder class for "sp_weights::weight_v2::Weight" not found

### ISSUE THREAD

**Issue by stepanLav** [polkascan/py-substrate-interface#274]
Hello!
For some networks, when I try to `substrate.get_payment_info(call=call, keypair=test_keypair)` it falls down with error:
`Decoder class for "sp_weights::weight_v2::Weight" not found`

Shortest code to reproduce it:
```python
from substrateinterface import SubstrateInterface, Keypair

url = "wss://karura-rpc.dwellir.com"

substrate = SubstrateInterface(url=url)
test_keypair = Keypair.create_from_uri('//Alice')
call = substrate.compose_call(
    call_module='Balances',
    call_function='transfer',
    call_params={
        'dest': test_keypair.ss58_address,
        'value': 0,
    }
)
substrate.get_payment_info(call=call, keypair=test_keypair)
```

Environment:
❯ python --version
Python 3.11.0
substrate-interface==1.4.0

**Comment by arjanz** [comment]
Ok this seems to be a backwards compatibility issue for older runtimes, please downgrade to [v1.3.5](https://github.com/polkascan/py-substrate-interface/releases/tag/v1.3.5) until this is fixed.

**Comment by stepanLav** [comment]
> Ok this seems to be a backwards compatibility issue for older runtimes, please downgrade to [v1.3.5](https://github.com/polkascan/py-substrate-interface/releases/tag/v1.3.5) until this is fixed.

It seems does not work for me, my steps are:
```
change to substrate-interface==1.3.5
❯ rm -r .venv/   
❯ python3 -m venv .venv     
❯ pip install -r requirements.txt      
```
on 1.3.5 it also falls with "Decoder class for "sp_weights::weight_v2::Weight" not found", maybe I can add it manually somehow?

**Comment by arjanz** [comment]
Otherwise you can try to add a type with that name manually to the type registry, but it needs some knowledge of custom types

**Comment by arjanz** [comment]
Will be available in next release, after merging another feature

**Comment by arjanz** [comment]
Released in [v1.4.1](https://github.com/polkascan/py-substrate-interface/releases/tag/v1.4.1).
